[Docs] Add Prisma Pulse comparison (#705)
sequinstream/sequin: docs/compare.mdx

3. **Scale**: LISTEN/NOTIFY is best suited for simple, low-volume use cases. Sequin is built to handle high-throughput scenarios with multiple consumers.

## Sequin vs Prisma Pulse

> [Prisma Pulse](https://www.prisma.io/docs/pulse) is a managed change data capture (CDC) service that's part of the Prisma ecosystem. While both tools enable real-time database change subscriptions, they differ in several ways:

Reviewer: ```suggestion
[Prisma Pulse](https://www.prisma.io/docs/pulse) is a managed change data capture (CDC) service that's part of the Prisma ecosystem. While both Prisma Pulse and Sequin enable real-time database change subscriptions, they differ in several ways:
```

1. **Integration model**: Prisma Pulse is tightly integrated with Prisma ORM and requires using Prisma Client (which only supports JavaScript). Sequin works with any programming language or framework.

> 2. **Destination support**: Sequin streams directly to multiple destinations (Kafka, SQS, Redis Streams, HTTP) and as a result can be used for a wider range of use cases. Prisma Pulse works only with the Prisma SDK and ORM and is primarily designed for in-application features.

Reviewer: Huh? This doesn't explain what the destination is – is it a JS callback? Webhook?

> 3. **Infrastructure**: Prisma Pulse is a fully managed service requiring a Prisma Data Platform account and a subscription. Sequin can be self-hosted as a Docker container or used as a managed service via Sequin Cloud.

Reviewer: I'd lean into the ability to run locally more

3. **Scale**: LISTEN/NOTIFY is best suited for simple, low-volume use cases. Sequin is built to handle high-throughput scenarios with multiple consumers.

## Sequin vs Prisma Pulse

[Prisma Pulse](https://www.prisma.io/docs/pulse) is a managed change data capture (CDC) service that's part of the Prisma ecosystem. While both Prisma Pulse and Sequin enable real-time database change subscriptions, they differ in several ways:

1. **Integration model**: Prisma Pulse is tightly integrated with Prisma ORM and requires using Prisma Client (which only supports JavaScript). Sequin works with any programming language or framework.

2. **Destination support**: Sequin streams directly to multiple destinations (Kafka, SQS, Redis Streams, HTTP) and as a result can be used for a wider range of use cases. Prisma Pulse delivers changes to the Prisma SDK / ORM via a black-box streaming mechanism and Javascript callbacks in the client. It is primarily designed for in-application features.

> 3. **Guarantees**: Sequin provides exactly-once processing with automatic retries and strict ordering that you can configure. Prisma Pulse offers options for exactly-once processing and ordering based on when the database change was detected when persistence is enabled.

Reviewer: ```suggestion
3. **Guarantees**: Sequin provides exactly-once processing with automatic retries and strict ordering that you can configure. You can enable exactly-once processing with Prisma Pulse by turning on persistence.
```
